app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -56,11 +56,61 @@ export const NOTES_LIST_DATA = [
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assigned_contact: null,
+  tags: [],
 };
 
+export const ASSIGNED_CONTACTS = [
+  {
+    label: "Value One",
+    value: "value1",
```

Comment:
Keep the values of label and value associated. Keep the `value` as the camelCased version of `label`.